### PR TITLE
Handle `file` errors

### PR DESCRIPTION
**Pull request by jtbandes**
The `file` command may sometimes fail. For example, certain versions of it fail on `.wasm` files: https://bugs.astron.com/view.php?id=170

It's only used for determining if a file is a Mach-O file, so it seems like errors can be safely ignored here.

This PR makes it possible to package an app that contains a wasm file with asar disabled.

### CODE REVIEW

**Comment by jtbandes** [comment]
Oops, I just realized @erkyrath already did this in #13.

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   "scripts": {
     "build": "tsc && tsc -p tsconfig.esm.json",
     "lint": "prettier --check \"src/**/*.ts\"",
-    "prepublishOnly": "npm run build",
+    "prepack": "npm run build",
```
Comment (MarshallOfSound) [inline package.json:25]: Is this change required?

Comment (jtbandes) [reply package.json:25]: This was only required in order to be able to `yarn install` directly from this fork. The `newLine` setting was in order to avoid inconsistent output and checksum mismatches on Windows.

However, after making this PR I realized there was an older one which addresses the same `file` issue. Please take a look: #13
